docs/user_docs/guides/working-with-k8s/1-adopt-from-kubernetes.md

```diff
@@ -47,3 +56,7 @@ If you developed CRDs, you can generate the KCL version of the CRD schemas and d
 * Define CR based on CRDs in KCL
 
     You can initialize the CRD schema to define a CR, or further, you can use the generated schema as a backend model and design a frontend interface for users to initialize. The practice is similar to what `KCL Models` does on Kubernetes built-in models.
+
+## Summary
+
+This section provides a quick start guide for using KCL with OpenAPI and Custom Resource Definitions (CRD). KCL also supports OpenAPI through the `kcl-openapi tool`, which maps OpenAPI specifications to KCL language features.
```

Comment:
what does that mean:
KCL also supports OpenAPI through the `kcl-openapi tool`
Seems this guide is exactly about kcl supporting OpenAPI and CRD through kcl-openapi tool?